### PR TITLE
feat: Footer V2

### PR DESCRIPTION
**Pull request by diegonzs-roll**
## What's done
- Add Footer V2 component
## Tasks

- [ ] Have you written tests for new code (if applicable)?
- [x] Have you tested the changes on all the platforms?
  - [x] iOS
  - [x] Android
  - [x] Web
- [x] Have you added stories to demonstrate the new functionality (if there is any)?

## Issue
#117 

## Demo (screenshots, recordings, etc.)
https://www.loom.com/share/c54b88f585b745939bc2586c85efdc8e

### iOS
Screenshot 2023-02-01 at 4 58 45 AM

### Web Desktop
web-desktop

### Web Mobile
web-mobile

### CODE REVIEW

File: packages/design-system/src/molecules/footerV2/index.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+}) => {
+  const handlePress = useCallback(() => {
+    openLink(link, true)
+  }, [link])
+  return (
+    <View
+      mr={{ md: !isLast ? '4' : 0 }}
+      mb={{ base: !isLast ? '4' : 0, md: 0 }}
+    >
+      <Caption weight="bold" color="rgba(0, 0, 0, 0.3)" onPress={handlePress}>
```
Comment (diegonzs-roll) [inline packages/design-system/src/molecules/footerV2/index.tsx:33]: I'll add colors in a different PR. These are the new colors we are going to use on the design system.

File: packages/design-system/src/molecules/footerV2/index.tsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+          contentResponsiveStyles,
+          { borderTopColor: theme.background.silver },
+        ]}
+      >
+        <LogoIso />
+        <View style={[styles.linksContainer, linksContainerResponsiveStyles]}>
+          <View style={[socialContainerResponsiveStyles]}>
+            {social.map((item, index) => (
+              <FooterOption
+                key={`${item.title}-${index}`}
```
Comment (teimurjan) [inline packages/design-system/src/molecules/footerV2/index.tsx:119]: Can we avoid using `index` as a component key ([ref](https://dev.to/shiv1998/why-not-to-use-index-as-key-in-react-lists-practical-example-3e66))? I mean, there is very little chance of having repeated titles, right? Can we rely just on `item.title`?

File: packages/design-system/src/molecules/footerV2/index.tsx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+                isLast={index === social.length - 1}
+                {...item}
+              />
+            ))}
+          </View>
+        </View>
+        <View style={[tradeMarkResponsiveStyles]}>
+          <Caption weight="regular" color={theme.text.black[30]}>
+            ©2023 Roll
```
Comment (teimurjan) [inline packages/design-system/src/molecules/footerV2/index.tsx:137]:
```suggestion
            ©{new Date().getFullYear()} Roll
```

**Review by teimurjan — COMMENTED** [review]
Could you include the screenshots from all the platforms of the final UI, please?

**Review by teimurjan — APPROVED** [review]

**Pull request merged**